repository/src/main/java/org/alfresco/repo/action/ActionExecutionContext.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * #%L
+ * Alfresco Repository
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited

```

Comment:
Since int is a new class, are you sure that copyright should be from 2005? Just asking.

Reply:
Done